faunadb/query.py

```diff
@@ -324,19 +358,24 @@ def next_id():
   return _fn({"next_id": None})
 
 
-def database(db_name):
+def database(db_name, scope=None):
+  """See the `docs <https://fauna.com/documentation/queries#misc_functions>`__."""
+  return _params({"database": db_name}, {"scope": scope})
+
+
+def index(index_name, scope=None):
   """See the `docs <https://fauna.com/documentation/queries#misc_functions>`__."""
-  return _fn({"database": db_name})
+  return _params({"index": index_name}, {"scope": scope})
 
 
-def index(index_name):
+def class_expr(class_name, scope=None):
```

Comment:
Humm... not sure how I feel about `expr` sufix. Is it too bad to use just `class_`?

Reply:
That function was not created in this PR, I just added the `scope` parameter. But it follows the same pattern as `if_expr`, `lambda_expr`, `map_expr`, `filter_expr`, `and_expr`, `or_expr` and `not_expr`. I don't like that pattern either, if you want I can create another pr adding sibling functions like `if_`, `lambda_` and etc and deprecating the other ones.

Reply:
That'd be nice.